index.js

```diff
@@ -27,16 +27,36 @@
     hexBytes[i] = (i + 0x100).toString(16).substr(1);
   }
 
-  // Node & Browser support
   var _crypto;
-  if(typeof crypto !== 'undefined') {
-    _crypto = crypto;
+  setEnvironment();
+  function setEnvironment() {
```

Comment:
Could rename this to be more accurate. For example, "_setCrypto" -- as that's what this function is ultimately doing. I'd like to hear other thoughts, though.

Reply:
I don't see any value `setEnvironment()`. Let's remove that entirely.

You defined a function then called it. That's a job for an [IIFE](https://en.wikipedia.org/wiki/Immediately_invoked_function_expression), if you needed a closure... but you don't.

Reply:
Same goes for the other functions `isNodeEnvironment()` and `isBrowserEnvironment()`.

They are called once and one-liners. Maybe add comment above the code in a simple if tree.

Reply:
Doh! Removed all that to have a smaller changeset.